src/main/java/mods/eln/sixnode/electricalmath/ElectricalMathGui.java

```diff
@@ -65,7 +65,7 @@ protected void postDraw(float f, int x, int y) {
                 c = 0xFF108F00;
             else
                 c = 0xFFFF0000;
-            helper.drawString(8 + 44 / 2, 29, c, tr("%1$ Redstone(s) required", render.redstoneRequired));
+            helper.drawString(8 + 44 / 2, 29, c, tr("%1$ Redstone required", render.redstoneRequired));
```

Comment:
There should be a check so that when it is more than one it writes Redstones? [citation needed]

Reply:
Redstones sounds cursed.